src/schemas/21-686C/schema.js

```diff
@@ -98,6 +98,21 @@ let schema = {
   additionalProperties: false,
   definitions: _.merge(definitions,
     {
+      phone: {
+        type: 'string',
+        pattern: '^[-0-9]{10,12}$'
+      },
+      ssn: {
+        oneOf: [
+          {
+            type: 'string',
+            pattern: '^[0-9]{9}$'
+          }, {
+            type: 'string',
+            pattern: '^[0-9]{3}-[0-9]{2}-[0-9]{4}$'
```

Comment:
from what they said in the channel the api allows both types of ssn, how about we allow just the 9 digit ssn so it's consistent across our other forms?